README.md

```diff
@@ -42,6 +42,12 @@ The following [pluggable extras](dist/extras) are provided which can be dropped
 * [Named exports](dist/extras/named-exports.js) convenience extension support for global and AMD module formats (`import { x } from './global.js'` instead of `import G from './global.js'; G.x`)
 * [Named register](dist/extras/named-register.js) supports `System.register('name', ...)` named bundles which can then be imported as `System.import('name')` (as well as AMD named define support)
 * [Transform loader](dist/extras/transform.js) support, using fetch and eval, supporting a hookable `loader.transform`
+* [Import __useDefault](dist/extras/import-use-default.js) provides better interop with AMD modules who export a default value and rely on the
```

Comment:
Suggested description:

```
provides a convenience interop for AMD modules to return the direct AMD binding instead of a `{ default: amdModule }` object from `System.import`.
```

It's also nice to avoid the extra mentions of `__esModule` as I don't think this is necessary, and `__useDefault` is really more of an internal implementation detail which I'd even prefer not to mention too.

Reply:
👍 updated